src/include/daos/event.h

```diff
@@ -87,6 +87,10 @@ daos_event_complete(daos_event_t *ev, int rc);
 int
 daos_event_launch(struct daos_event *ev);
 
+/** convert event error to positive errno instead of -DER on completion */
```

Comment:
Maybe should say something like

Sets event so that positive errno instead of -DER is returned on completion

This text makes it look like this function should do the conversion